### ISSUE TITLE
IDE0071 invalid simplification

### ISSUE THREAD

**Issue by kindermannhubert** [dotnet/roslyn#43781]
**Version Used**: 
VS 16.5.4

```C#
class C
{
    public string P => $"{base.ToString()}";
}
```
is simplified by IDE0071 fixer to:
```C#
class C
{
    public string P => $"{base}";
}
```
which does not compile.

**Comment by sharwell** [comment]
Duplicate of #42669

**Comment by sharwell** [comment]
@kindermannhubert This should be fixed in 16.6 Preview 3 👍
